docs/architecture-decisions/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,19 @@
+# 1. Record architecture decisions
+
+Date: 2023-07-14
+
+## Status
+
+Proposed
```

Comment:
My plan is to change them to "Approved" when we get three 👍's. (If I were to do this over again, I would have made each of these a separate PR so we approve them individually.)

I'm sure @JPrevost is also 👍 on this, but I'll wait for explicit approval before making the change.

Reply:
Ah, yeah. I think I usually leave the ADRs as "approved" when they are in the PR with the assumption it makes them easier to merge once we actually approve them. I'm 👍🏻 on ADR 1 and 2 and they can be merged once we update them to approved. I'm not sure how best to do that without losing any of the dialog here so it may be best to just leave them unmerged even though we agree with them and continue working on 3 and 4 and aim towards merging them all in the next week or so.